.github/workflows/build.yml

```diff
@@ -10,11 +10,11 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
     steps:
```

Comment:
Do you know if there's a reason why there's no `defaults` block here as there is in other repositories?
``` defaults:
      run:
        shell: bash -le {0}
```

Reply:
I do not; I'll try adding it and making sure tests pass for consistency with other framework repos